pages/blog/event-driven-api-documentation-made-simple-clientside-rendering.md

```diff
@@ -0,0 +1,256 @@
+```bash
+npm install --save @asyncapi/react-component@next
+```
+
+2️⃣ Now, create a **index.js** file and type the following:
+
+```js
+import React from "react";
+import ReactDOM from "react-dom";
+
+import AsyncApiComponent from "@asyncapi/react-component";
+import "@asyncapi/react-component/styles/default.min.css";
+
+import { specMock } from "./testDoc";
+
+const rootElement = document.getElementById("root");
+ReactDOM.render(<AsyncApiComponent schema={specMock} />, rootElement);
+```
+
+Here we can see that the **React AsyncAPI component** is imported on **line 4**.
+
+If you are happy with AsyncAPI styling then you need to import their CSS pattern with:
+
+\```js
+import "@asyncapi/react-component/styles/default.min.css";
+\```
+
+**Line 7** is where the sample AsyncAPI file is imported.
+
+---
+
+### <a name="vue"></a>Vue
+
+If you wish to generate documentation from your AsyncAPI file in a Vue application, you need to use the [AsyncApiStandalone bundle](https://github.com/asyncapi/asyncapi-react/blob/next/docs/usage/vue.md).
+
+The given instructions are for Vue 2.x.
+
+1️⃣ The React AsyncAPI component is also required here, so you need to run the command
+```bash
+npm install --save @asyncapi/react-component@next
+```
+
+2️⃣ In your **App.vue** just add this code:
+
+```vue
+<template>
+  <div ref="asyncapi"></div>
+</template>
+
+<script>
+import AsyncApiStandalone from '@asyncapi/react-component/browser/standalone';
+
+// AsyncAPI specification, fetched or pasted.
+const schema =
+'{"asyncapi":"2.4.0","info":{"title":"Account Service","version":"1.0.0","description":"This service is in charge of processing user signups"},"channels":{"user/signedup":{"subscribe":{"message":{"$ref":"#/components/messages/UserSignedUp"}}}},"components":{"messages":{"UserSignedUp":{"payload":{"type":"object","properties":{"displayName":{"type":"string","description":"Name of the user"},"email":{"type":"string","format":"email","description":"Email of the user"}}}}}}}';
+
+const config = {}; // Configuration for component. This same as for normal React component.
+
+export default {
+  name: 'AsyncApiComponent',
+  props: {
+    msg: String
+  },
+  mounted() {
+    const container = this.$refs.asyncapi;
+    AsyncApiStandalone.render({ schema, config }, container);
+  }
+}
+</script>
+
+<style scope src="@/assets/asyncapi.min.css"></style>
+```
+
+As you can see on **line 6**, you need to import the **AsyncApiStandalone** bundle with command
+```vue
+import AsyncApiStandalone from '@asyncapi/react-component/browser/standalone';
+```
+
+3️⃣ If you like AsyncAPI styling, then there is one more thing to do. You can do it in few ways:
+
+◾ First way; you need to go to 👉 `node_modules/@asyncapi/react-component/style/default.min.css`.
+
+Copy that file and then paste it into your _**assets**_ folder.
+I renamed mine **asyncapi.min.css**.
+
+Then, you can import this in your **main.js** file with
+```js
+import './assets/asyncapi.min.css'
+```
+
```

Comment:
We can add here another section:

```suggestion
You can also import it at the end of the **App.vue** file with:

/```Vue
<style scope src='./assets/asyncapi.min.css'></style>
/```
```